helm-charts/vsts-agent/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A custom VSTS agent container with Python3, kubectl and msftkube preinstalled.
+name: vsts-agent
+version: 0.1.1094096
```

Comment:
Version and tag (in values.yaml) look very related. Do they need to change in tandem in these files?

Reply:
yes. This is the chart version. It helps the consumers to know what is the default image tag if they don't specify one